docs/source/GetStartedGuide.rst

```diff
@@ -211,6 +211,75 @@ keep reading.
           positional arguments, therefore test names (in case of
           ``avocado run``)
 
+Test References
+---------------
```

Comment:
Actually it's already described there and the detailed explanation here only applies to `FileLoader`. So how about tweak it a bit and create a `FileLoader discovery` section in `Loaders` with all these neat details?